lib/iris/tests/unit/lazy_data/test_co_realise_cubes.py

```diff
@@ -84,17 +81,7 @@ def test_combined_access(self):
         cube_b = Cube(derived_b)
         cube_c = Cube(derived_c)
         co_realise_cubes(cube_a, cube_b, cube_c)
-        # Though used more than once, the source data should only get fetched
```

Comment:
Having removed all the comments, I think it is no longer clear why "2" is the correct and appropriate answer !
(i.e. why didn't it just fetch once, isn't that exactly the point ??)
Could you add a comment about that ?
Also, to make it really clear, you could add in more simultaneous statistics, to show e.g. "I calculated 5 derived stats but it only fetched twice".